blog.md

```diff
@@ -0,0 +1,100 @@
+## Take a look at your repository from a new perspective with Git Machete Plugin!
+
+Git and IntelliJ-based software are currently one of the most important tools in software development.
```

Comment:
Hmmm just thinking now... is this paragraph even needed in the VL blog? :thinking: could we start from `Creating and maintaining...`?